Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/lucene/luceneindex.jsp

```diff
@@ -18,11 +18,10 @@
 <script type="text/javascript">
 	$(document).ready(function() {
 		var lastRun = $('#lastRun').val();
-		var d = new Date(lastRun);
-		setDateTime(d.toLocaleString());
+		setDateTime(lastRun);
```

Comment:
Sorry I forgot about the different timezone.  I included again since it wont work correctly on different timezone.